src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -110,12 +110,12 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ErrorInfo'
+                $ref: '../../../../common-rest-api/src/main/resources/common-openapi-definition.yaml#/components/schemas/ErrorInfo'
```

Comment:
It would probably be better to introduce local schema object `ErrorInfo` which refs to common-rest-api/ErrorInfo. This way the long relative path to common-openapi-definition does not need to be repeated in so many places, and we have easier time if folder structure or naming needs to be updated (and it's a bit more clear for the reader as well I guess)

Like here for `User`: https://github.com/nordic-institute/X-Road/blob/XRDDEV-1715-importMappings/src/proxy-ui-api/src/main/resources/openapi-definition.yaml#L6876-L6877

It seemed like the end result (= generated source code) is not affected by this(?)